### PR TITLE
Manually add order in workshop

### LINKED ISSUE #493: [imp] add order via worskhop to a production day
Make it possible for bakers to add an order in the workshop to a production day.

### PR DESCRIPTION
**Pull request by bruecksen**
fixes #493

### CODE REVIEW

File: bakeup/templates/workshop/customerorder_form.html [hunk 1]
```diff
@@ -2,20 +2,31 @@
 {% load i18n %}
 {% load django_bootstrap5 %}
 
-{% block title %}{% trans "Add new production day" %}{% endblock %}
+{% block title %}{% trans "Add neworder" %}{% endblock %}
```
Comment (madmooose) [inline bakeup/templates/workshop/customerorder_form.html:5]: Whitespace missing?

File: bakeup/templates/workshop/customerorder_form.html [hunk 2]
```diff
@@ -2,20 +2,31 @@
...
 {% block page_heading %}
-{% if object %}{% trans "Update customer order" %}{% else %}{% trans "Add new production day" %}{% endif %}
+{% if object %}{% trans "Update customer order" %}{% else %}{% trans "Add order" %}{% endif %}
```
Comment (madmooose) [inline bakeup/templates/workshop/customerorder_form.html:7]: New or not New isn't tgis the same AS above

File: bakeup/templates/workshop/productionday_detail.html [hunk 3]
```diff
@@ -76,7 +76,8 @@ <h2 class="mt-4">{% trans "Summary" %}
     </tr>
       {% endfor %}
     </table>
-    <a href="{% url 'workshop:order-add' pk=object.pk %}?next={{ object.get_absolute_url }}" class="btn btn-primary btn-sm float-end">{% trans "Edit orders" %}</a>
+    <a href="{% url 'workshop:order-batch' pk=object.pk %}?next={{ object.get_absolute_url }}" class="btn btn-primary btn-sm float-end">{% trans "Edit orders" %}</a>
+    <a href="{% url 'workshop:order-add' production_day=object.pk %}" class="btn btn-primary btn-sm float-end me-2"><i class="fas fa-plus-circle"></i> {% trans "Add order" %}</a>
```
Comment (madmooose) [inline bakeup/templates/workshop/productionday_detail.html:80]: New or not

File: bakeup/workshop/views.py [hunk 4]
```diff
@@ -783,45 +786,64 @@ def get_context_data(self, **kwargs):
...
-    fields = ["point_of_sale"]
-    template_name = "workshop/customerorder_form.html"
-    success_url = reverse_lazy("workshop:order-list")
+class CustomerOrderCreationMixin(object):
+    extra_formset = 0
+
+    def get(self, request, *args, **kwargs):
+        self.object = self.get_object()
+        if self.get_production_day().has_production_plan_started:
+            messages.add_message(
```
Comment (madmooose) [inline bakeup/workshop/views.py:795]: Shouldn't we mention the product here

File: bakeup/templates/workshop/customerorder_form.html (hunk 2 above)
Comment (bruecksen) [reply bakeup/templates/workshop/customerorder_form.html:7]: yes it is inconsistent all over the place :| Need to fix this at one point. Thanks for feedback!

**Pull request merged**